release-calendar/src/types.ts

```diff
@@ -26,6 +26,20 @@ export enum Channel {
   CREATED = 'created',
 }
 
+interface Inames {
+  [key: string]: string;
+}
+
+export const channelNames: Inames = {
```

Comment:
I'm generally ok with keeping this in `types` but this also works somewhere in the client side, since the server won't need this.

Why not combine this with `ChannelTable.channels`?